### PR TITLE
[build] add `sdk_support` submodule in place of `gecko_sdk_suite`

### PR DESCRIPTION
**Pull request by lmnotran**
This replaces the `gecko_sdk_suite`, which the user used to have to install manually, with [SiliconLabs/sdk_support](https://github.com/SiliconLabs/sdk_support).

It also adds a step to `./script/build` to generate .s37 images after a successful cmake build. These images are what's used to flash using Simplicity Studio Commander

### CODE REVIEW

File: src/efr32mg12/README.md [hunk 1]
```diff
@@ -248,7 +207,7 @@ The above example demonstrates basic OpenThread capabilities. Enable more featur
 ```bash
 $ cd <path-to-ot-efr32>
 $ ./bootstrap
```
Comment (romacdon) [inline src/efr32mg12/README.md:209]: Shouldn't this be ./script/bootstrap?

File: src/efr32mg1/README.md [hunk 2]
```diff
@@ -233,7 +199,7 @@ The above example demonstrates basic OpenThread capabilities. Enable more featur
 ```bash
 $ cd <path-to-ot-efr32>
 $ ./bootstrap
```
Comment (romacdon) [inline src/efr32mg1/README.md:201]: Shouldn't this be ./script/bootstrap?

File: src/efr32mg13/README.md [hunk 3]
```diff
@@ -247,7 +206,7 @@ The above example demonstrates basic OpenThread capabilities. Enable more featur
 ```bash
 $ cd <path-to-ot-efr32>
 $ ./bootstrap
```
Comment (romacdon) [inline src/efr32mg13/README.md:208]: Shouldn't this be ./script/bootstrap?

File: src/efr32mg21/README.md [hunk 4]
```diff
@@ -191,7 +152,7 @@ The above example demonstrates basic OpenThread capabilities. Enable more featur
 ```bash
 $ cd <path-to-efr32>
 $ ./bootstrap
```
Comment (romacdon) [inline src/efr32mg21/README.md:154]: Shouldn't this be ./script/bootstrap?

**Review by romacdon — CHANGES_REQUESTED** [review]

File: .gitmodules [hunk 5]
```diff
@@ -1,3 +1,6 @@
 [submodule "openthread"]
 	path = openthread
 	url = https://github.com/openthread/openthread.git
+[submodule "third_party/silabs/sdk_support"]
+	path = third_party/silabs/sdk_support
+	url = git@github.com:SiliconLabs/sdk_support.git
```
Comment (jwhui) [inline .gitmodules:6]: Change to `https`?
```suggestion
	url = https://github.com/SiliconLabs/sdk_support.git
```

File: src/efr32mg1/README.md (hunk 2 above)
Comment (jwhui) [reply src/efr32mg1/README.md:201]: Is `bootstrap` even required? Can we remove this line?

File: src/sleepy-demo/README.md [hunk 6]
```diff
@@ -9,20 +9,19 @@ For setting up the build environment refer to [examples/platforms/efr32/README.m
 In this `README`, all example commands will be targeting the `efr32mg12` platform. The same commands should work for the other `efr32` platforms
 
 ```bash
-$ cd <path-to-openthread>
-$ ./bootstrap
-$ make -f examples/Makefile-efr32mg12 COMMISSIONER=1 JOINER=1 DHCP6_CLIENT=1 DHCP6_SERVER=1 BOARD=BRD4161A
+$ cd <path-to-ot-efr32>
+$ ./script/bootstrap
```
Comment (romacdon) [inline src/sleepy-demo/README.md:13]: Remove?
```suggestion
```

**Review by romacdon — CHANGES_REQUESTED** [review]

**Comment by romacdon** [comment]
@googlebot I fixed it

**Review by romacdon — APPROVED** [review]
LGTM

**Comment by jwhui** [comment]
@lmnotran , it appears a couple of your commits have a different email address, which is why the CLA checker is blocking. Can you amend the commits to be consistent?

**Comment by romacdon** [comment]
@googlebot I fixed it

**Comment by romacdon** [comment]
> @lmnotran , it appears a couple of your commits have a different email address, which is why the CLA checker is blocking. Can you amend the commits to be consistent?

@jwhui we were thinking the issue was with me being a co-author of Mason's last commit and that my email was not public and therefore the CLA was not passing.  I have since made my email public and also added 54455546+romacdon@users.noreply.github.com to our corporate CLA group but that didn't fix it.  I'm not sure what to do.

**Comment by jwhui** [comment]
> @jwhui we were thinking the issue was with me being a co-author of Mason's last commit and that my email was not public and therefore the CLA was not passing. I have since made my email public and also added [54455546+romacdon@users.noreply.github.com](mailto:54455546+romacdon@users.noreply.github.com) to our corporate CLA group but that didn't fix it. I'm not sure what to do.

Some commits have:
```
commit 34f3403164e4ccf0e992b74c07f442d18a0906ca
Author: Mason Tran <mason@lmnotran.com>
Date:   Tue Mar 30 13:15:19 2021 -0400
```

While other commits have:
```
commit 1934552da91486ed290b12002a3dd20004ed2304
Author: Mason Tran <Mason.Tran@silabs.com>
Date:   Tue Mar 30 10:37:11 2021 -0400
```

I presume @lmnotran should always use the latter email address in his commits.

**Comment by lmnotran** [comment]
@jwhui @romacdon All good. Sorry about the mess! 😅

**Review by jwhui — APPROVED** [review]

**Pull request merged**